node/packages/sdk/.ts-types/index.d.ts

```diff
@@ -7,41 +7,45 @@ export interface Instrumentation {
   expressApp: ExpressAppInstrument;
 }
 
-interface Sdk {
+export function createTraceSpan(
+  name: string,
+  options?: {
+    startTime?: bigint;
+    immediateDescendants?: string[];
+    tags?: Record<string, boolean | number | string | Date | Array<unknown> | null>;
+    input?: string;
+    output?: string;
+    onCloseByRoot?: Function;
+  }
+): TraceSpan;
+
+export function captureError(
+  error: Error,
+  options?: {
+    fingerprint?: string;
+    tags?: Record<string, boolean | number | string | Date | Array<unknown> | null>;
+  }
+): undefined;
+
+export function captureWarning(
+  message: string,
+  options?: {
+    fingerprint?: string;
+    tags?: Record<string, boolean | number | string | Date | Array<unknown> | null>;
+  }
+): undefined;
+
+export interface Sdk {
   name: string;
   version: string;
   orgId: string;
   traceSpans: TraceSpans;
   instrumentation: Instrumentation;
```

Comment:
Wouldn't the `Sdk` interface still need those method definitions on them @Danwakeem ?

Reply:
@Mmarzex yes, I believe that is what the next three lines should do 👇 

https://github.com/serverless/console/blob/0444c35962983cfb69056c977e3ebf5bab672d53/node/packages/sdk/.ts-types/index.d.ts#L44-L46

Reply:
🤦 Totally missed that.